docs/cluster-configuration.md

```diff
@@ -23,7 +23,7 @@ cluster = Cluster(ClusterConfiguration(
     labels={"exampleLabel": "example", "secondLabel": "example"},
 ))
 ```
-Note: 'quay.io/rhoai/ray:2.23.0-py39-cu121' is the default community image used by the CodeFlare SDK for creating a RayCluster resource. If you have your own Ray image which suits your purposes, specify it in image field to override the default image.
```

Comment:
Can we add a comment about available ROCM images for AMD GPUs too? that way users can easily find them?

Reply:
I've updated this, please let me know if its sufficient. These 2 images are available from the quay.io/modh org, the other torch ones are [here](https://quay.io/repository/rhoai/ray?tab=tags), should I add reference to those also?

Reply:
I think linking would be good. Something like
"A list of available images can be found here..."

Reply:
@astefanutti Do you think it is okay to link to these [images](https://quay.io/repository/rhoai/ray?tab=tags) here? Or we should wait until we have documented them with an explanation as to their compatibility etc?

Reply:
@Fiona-Waters I think that's OK to mention the `quay.io/modh/ray:2.23.0-py39-rocm61` image as @KPostOffice suggested. We can state AMD accelerators support is dev preview at the moment if you think that's important to mention in upstream documentation.

I would not mention example images from `quay.io/rhoai`. Alternatively, we could point to the documentation on how to build custom images, linking to https://github.com/opendatahub-io/distributed-workloads/blob/main/images/runtime/examples/README.md.

Reply:
Ok will link to the modh rocm image and to that documentation. Thank you!

Reply:
@KPostOffice if you are happy with this we can resolve this conversation and merge. Thank you!